ttml.go

```diff
@@ -421,6 +432,15 @@ func ReadFromTTML(i io.Reader) (o *Subtitles, err error) {
 			}
 
 		}
+		//if multiple items in `l` are present without new line then adding space at end so it can become readable after concatenation
+		if len(l.Items) > 0 {
```

Comment:
I think if we implement the rule `trim spaces on input, add in-between spaces on output`, we don't need this logic anymore.

However you can add this logic on output [here](https://github.com/asticode/go-astisub/blob/master/ttml.go#L643) to make sure we add `lineItem.Text + ""` when `idx < len(item.Lines) -1`